step/step.go

```diff
@@ -0,0 +1,102 @@
+package step
+
+import (
+	"fmt"
+	"github.com/bitrise-io/go-steputils/v2/stepconf"
+	"github.com/bitrise-io/go-utils/v2/env"
+	"github.com/bitrise-io/go-utils/v2/log"
+	"github.com/bitrise-steplib/tmp-bitrise-step-create-new-app-store-release/api"
+	"github.com/bitrise-steplib/tmp-bitrise-step-create-new-app-store-release/export"
+)
+
+const (
+	releaseUrlKey  = "BITRISE_RELEASE_URL"
+	releaseSlugKey = "BITRISE_RELEASE_SLUG"
+)
+
+type ReleaseExecutor struct {
+	inputParser   stepconf.InputParser
+	envRepository env.Repository
+	logger        log.Logger
+}
+
+func NewReleaseExecutor(inputParser stepconf.InputParser, envRepository env.Repository, logger log.Logger) ReleaseExecutor {
+	return ReleaseExecutor{
+		inputParser:   inputParser,
+		envRepository: envRepository,
+		logger:        logger,
+	}
+}
+
+func (r ReleaseExecutor) ProcessConfig() (Config, error) {
+	var input Input
+	err := r.inputParser.Parse(&input)
+	if err != nil {
+		return Config{}, err
+	}
+
+	stepconf.Print(input)
+	r.logger.Println()
+	r.logger.EnableDebugLog(input.Verbose)
+
+	return Config{
+		BitriseApiBaseUrl:         input.BitriseApiBaseUrl,
+		BitriseApiAccessToken:     string(input.BitriseApiAccessToken),
```

Comment:
Note: it is safer to store this property as a stepconf.Secret and cast to string only when using. This way we can avoid accidentally logging it.